common/changes/@itwin/viewer-react/leck-localization_2022-07-05-19-10.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "Added custom localization option",
+      "type": "patch"
```

Comment:
nbd, and my fault i didnt pay closer attention, but this shouldve been a minor bump.

but nbd

Reply:
Oops, I always have trouble discerning between minor vs patch. I'll pay more attention to it next time.... minor is adding a new feature and patch is fixing a bug, right?